bin/render_source.jl

```diff
@@ -0,0 +1,55 @@
+using Celeste:
+    Model, DeterministicVI, CelesteEDA, DeterministicVIImagePSF, SDSSIO,
+    Infer, PSF
+
+# NB: PyPlot is not in REQUIRE.
+using PyPlot
+
+##########
+# Load data.
+
+const datadir = joinpath(Pkg.dir("Celeste"), "test", "data")
+rcf = SDSSIO.RunCamcolField(4263, 5,119)
+images = SDSSIO.load_field_images(rcf, datadir);
+catalog = SDSSIO.read_photoobj_files([rcf], datadir, duplicate_policy=:first);
+
+# Pick an object.
+for cat in catalog
+    if minimum(cat.star_fluxes) > 300
+        print(cat.objid)
+        print("\n")
+    end
+end
+objid = "1237663784734490677"
+objids = [ce.objid for ce in catalog];
+sa = findfirst(objids, objid);
+neighbors = Infer.find_neighbors([sa], catalog, images)[1];
+cat_local = vcat(catalog[sa], catalog[neighbors]);
+vp = Vector{Float64}[Infer.init_source(ce) for ce in cat_local];
+patches = Infer.get_sky_patches(images, cat_local);
+
+
+###########################
+# Visualize a source at initial parameters using both classic and FFT Celeste.
+
+ea = ElboArgs(images, deepcopy(vp), patches, [1]);
+Celeste.Infer.load_active_pixels!(ea);
+
+ea_fft = ElboArgs(images, deepcopy(vp), patches, [1], psf_K=1);
+Celeste.Infer.load_active_pixels!(ea_fft, exclude_nan=false);
```

Comment:
The way the PSF fitting code works for galaxies is that it first renders the object in the active pixels and then convolves with the PSF using a FFT.  If the active pixels have a big hole in the middle (as is usually the case for bright objects), then the pre-convolution object is hardly rendered at all.

The right solution is to have a separate set of active pixels for pre- and post-convolution rendering.  I'll leave it to you to decide how to handle that, though.